src/main.rs

```diff
@@ -207,12 +207,18 @@ fn safe_remove_tree(path : &str) -> Result<()> {
     Ok(())
 }
 
-fn destroy(container_name : &str) -> Result<()> {
+fn stop(container_name : &str) -> Result<()> {
     match run(Command::new("systemctl").arg("stop").arg(format!("container@{}", container_name))) {
-        Ok(()) => {},
-        Err(Interrupted(_)) => { let _ = run::<()>(Command::new("systemctl").arg("kill").arg(format!("container@{}", container_name))); },
-        e => return e
+        Ok(()) => Ok(()),
+        Err(Interrupted(_)) => {
+            return run(Command::new("systemctl").arg("kill").arg(format!("container@{}", container_name)))
```

Comment:
The result of the kill command was deliberately not returned because I want to clean up unconditionally after receiving SIGINT/SIGTERM.